game_sessions/repository/postgres/game_session.go

```diff
@@ -105,6 +109,50 @@ func (r *GameSessionsPostgresRepo) GetGameSession(ctx context.Context, id uint64
 	return toModelGameSession(session)
 }
 
+func (r *GameSessionsPostgresRepo) GetGlobalSessions(ctx context.Context, filter gamesessions.FilterType) ([]*models.GameSession, error) {
+	conn, err := db.DbPool.Acquire(ctx)
+	if err != nil {
+		return nil, err
+	}
+	defer conn.Release()
+
+	var rows pgx.Rows
+	if filter == gamesessions.All {
+		rows, err = conn.Query(ctx, selectGlobalSessionsStmt, models.GameFinished, 30)
+		if err != nil {
+			return nil, err
+		}
+	} else if filter == gamesessions.Wins {
+		rows, err = conn.Query(ctx, selectGlobalSessionsWinsStmt, models.GameFinished, 30)
+		if err != nil {
+			return nil, err
+		}
+	} else if filter == gamesessions.Losts {
+		rows, err = conn.Query(ctx, selectGlobalSessionsLostStmt, models.GameFinished, 30)
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		return nil, errors.New("bad filter")
+	}
+
+	gameSessions := make([]*models.GameSession, 0)
```

Comment:
You know max count rows - 30, use make([]*models.GameSession, 0, 30); And 30 - magic variable! use constatnt